webview/src/plots/components/styles.module.scss

```diff
@@ -58,6 +58,33 @@ $gap: 20px;
   }
 }
 
+.plotActions {
+  z-index: 500;
+  width: 25px;
```

Comment:
I aimed for a size that looked "ok" across the different plot sizes since it's easier than adjust the sizing based off the one of the multiple plots size variations.